### DISCUSSION TITLE
Deploy Django with Granian

### DISCUSSION THREAD

**Discussion by JaimeOnaindia** [emmett-framework/granian#187] [Q&A]
It is posible? It's there any example? 
Thanks

**Comment by gi0baro** [comment]
It really depends on your deployment context, but in general it is usually just about changing the command used to start the application to use granian, eg:

```
# WSGI
granian --interface wsgi project.wsgi:application
# ASGI
granian --interface asgi project.asgi:application
```

Running `granian --help` will provide you all the possible options.

**Reply by MIKNOTAURO** [reply]
this would be the replacement for daphne and nginx?
I meant... a production web server & application server?

**Reply by gi0baro** [reply]
> this would be the replacement for daphne and nginx? I meant... a production web server & application server?

Depends on your needs, but potentially yes.
If you need to serve several applications on the same machine though, you would still need a proxy like nginx to route traffic to the various applications.

**Comment by JaeHyuckSa** [comment]
Hi! @gi0baro   I’ve also posted a related proposal on the Django community forum to suggest adding Granian to the official “How to deploy” documentation (ASGI & WSGI support).
You can find it here:
- https://forum.djangoproject.com/t/add-granian-to-how-to-deploy-docs-asgi-wsgi-support/43386
